### ISSUE TITLE
[CI outer loop] SendPingToExternalHostWithLowTtlTest

### ISSUE THREAD

**Issue by benaadams** [dotnet/corefx#35332]
Skipped on all platforms bar the failing ones

OSX.1013.Amd64.Open-x64-Debug
```
Expected: Not Success
Actual:   Success

at System.Net.NetworkInformation.Tests.PingTest.SendPingToExternalHostWithLowTtlTest() in
/Users/vsts/agent/2.146.0/work/1/s/src/System.Net.Ping/tests/FunctionalTests/PingTest.cs:line 812
--- End of stack trace from previous location where exception was thrown ---
```

OSX.1012.Amd64.Open-x64-Debug
```
Expected: Not Success
Actual:   Success

at System.Net.NetworkInformation.Tests.PingTest.SendPingToExternalHostWithLowTtlTest() in
/Users/vsts/agent/2.146.0/work/1/s/src/System.Net.Ping/tests/FunctionalTests/PingTest.cs:line 812
--- End of stack trace from previous location where exception was thrown ---
```

https://mc.dot.net/#/user/dotnet-bot/pr~2Fdotnet~2Fcorefx~2Frefs~2Fpull~2F35234~2Fmerge/test~2Ffunctional~2Fcli~2F~2Fouterloop~2F/20190214.1/workItem/System.Net.Ping.Functional.Tests/analysis/xunit/System.Net.NetworkInformation.Tests.PingTest~2FSendPingToExternalHostWithLowTtlTest

**Comment by wfurt** [comment]
I've been trying to investigate this. It seems like environment issue and it is pretty consistent.

**Comment by wfurt** [comment]
should be fixed by #35506
